src/Cake.Core/Extensions/ProcessArgumentListExtensions.cs

```diff
@@ -380,5 +380,39 @@ public static ProcessArgumentBuilder AppendQuotedSecret(this ProcessArgumentBuil
             }
             return builder;
         }
+
+        /// <summary>
+        /// Indicates whether a <see cref="ProcessArgumentBuilder"/> is null or renders empty.
```

Comment:
Unsure what you mean :)
![image](https://cloud.githubusercontent.com/assets/1647294/15837190/80f9d504-2c38-11e6-9641-a1783ecc2b82.png)